### PR TITLE
Added new Blue Wreck sizes and loot to said wrecks

### PR DESCRIPTION
**Pull request by wafehling**
## About the PR
In this PR, I:

-Rebalanced the size of blue wrecks to be larger
-Added a variety of additional loot to them, such as more crates, furniture, tanks, etc.

This is my first properly-sized PR, feedback/criticism is encouraged! Nothing's set in stone.

## Why / Balance
Salvaging pays far worse than regular old mining does, and is a more cumbersome and difficult task, and the loot pool lacks many basic things. 

This aims to reduce the value/fun gap between the two activities and make salvaging more useful, varied, and interesting.

## Technical details
All I've done is add the new spawns and wreck sizes to the yaml, no deeper code changes have been made. 

## Media

Here are some shots of the new larger sizes of wreck (They don't all spawn this big, don't worry) and some of the new loot/structure you can find on them.

radar size comparison
large wreck shot 1
large wreck shot 2
large wreck shot 3

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
Shouldn't break anything! Testing was smooth as silk, and nothing appears to be broken, and I limited my changes as much as I could.

**Changelog**
add: New sizes of blue salvage wrecks
add: New variety of loot to blue salvage wrecks
tweak: some spawn chances for loot on wrecks

### CODE REVIEW

**Comment by Cheackraze** [comment]
thank you this has been sorely needed, however, we are about to do a really big upstream merge, so this may be put on pause until then.

The most important takeway from that, however, is that in about a week's time, there will be *even more* things you can add to this loot pool!

**Comment by Cheackraze** [comment]
The mergening has happened, feel free to add more items to the loot table list. Things like the roller skates and handheld flags and all of that neato stuff we got, way too much to type out tbh. I must say, super excited for this to finally spice up the salvage experience

**Comment by wafehling** [comment]
> The mergening has happened, feel free to add more items to the loot table list. Things like the roller skates and handheld flags and all of that neato stuff we got, way too much to type out tbh. I must say, super excited for this to finally spice up the salvage experience

Sure! Is there a comprehensive list of what was added?

**Comment by Cheackraze** [comment]
> > The mergening has happened, feel free to add more items to the loot table list. Things like the roller skates and handheld flags and all of that neato stuff we got, way too much to type out tbh. I must say, super excited for this to finally spice up the salvage experience
> 
> Sure! Is there a comprehensive list of what was added?

its about 3 months worth of items from upstream, theres a lot haha. 
Some suggestions:

- kobold cube crate
- handheld flags (theres several)
- crossbow stuff
- rollerskates
- candles
- various new plushies

im sure theres a bunch more i missed too, but let me know when your list is finalized and we can get this in

**Comment by wafehling** [comment]
> > > The mergening has happened, feel free to add more items to the loot table list. Things like the roller skates and handheld flags and all of that neato stuff we got, way too much to type out tbh. I must say, super excited for this to finally spice up the salvage experience
> > 
> > 
> > Sure! Is there a comprehensive list of what was added?
> 
> its about 3 months worth of items from upstream, theres a lot haha. Some suggestions:
> 
> * kobold cube crate
> * handheld flags (theres several)
> * crossbow stuff
> * rollerskates
> * candles
> * various new plushies
> 
> im sure theres a bunch more i missed too, but let me know when your list is finalized and we can get this in

Alright! I believe I'm finished for now. Just sent a push with a few of the new items added. I'll make a new PR when I come up with a list of further things to add, but right now this is a huge increase in variety already.

Thanks a ton!

File: Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml [hunk 1]
```diff
@@ -31,12 +31,47 @@
...
+        - CrateFunMysteryFigurines
+        - CrateFunDartsSet
+        - CrateMedicalSupplies
+        - CrateChemistrySupplies
+        - CrateEmergencyBurnKit
+        - CrateEmergencyToxinKit
+        - CrateEmergencyO2Kit
+        - CrateEmergencyBruteKit
+        - CrateEmergencyAdvancedKit
+        - CrateEmergencyRadiationKit
```
Comment (Cheackraze) [inline Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml:63]: The medical crates are the only thing I have an issue with.  Having a place for medical ships is already a struggle, and having meds be this easy to find and collect only makes the problem worse.

**Review by Cheackraze — CHANGES_REQUESTED** [review]
The medical crates need removing, specifically the pre-made medicine ones (the chemistry crates can stay). Once that is done, I can help fix whatever issues the yml winter is having and we can get this merged

File: Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml (hunk 1 above)
Comment (wafehling) [reply Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml:63]: That's more than fair!

**Comment by wafehling** [comment]
> The medical crates need removing, specifically the pre-made medicine ones (the chemistry crates can stay). Once that is done, I can help fix whatever issues the yml winter is having and we can get this merged

Yeah, I'm not sure what I'm doing when it comes to the sprites section of the yml.

**Comment by Cheackraze** [comment]
and dont forget to remove the emergency medical crates as well, since it undermines the players who play doctors

File: Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml (hunk 1 above)
Comment (wafehling) [reply Resources/Prototypes/Entities/Markers/Spawners/Random/salvage.yml]: Ahh. Didn't realize those didn't get automatically removed when I resolved the change. Removed em now!

**Comment by wafehling** [comment]
> and dont forget to remove the emergency medical crates as well, since it undermines the players who play doctors

Didn't realize those didn't get automatically removed when I resolved the change. Removed em now!

**Review by Cheackraze — APPROVED** [review]

**Comment by wafehling** [comment]
Hooray!

**Pull request merged**